client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notification-settings/form-content/email-notification.tsx

```diff
@@ -22,11 +24,12 @@ export default function EmailNotification( {
 	defaultUserEmailAddresses,
 	toggleAddEmailModal,
 	allEmailItems,
+	restriction,
 }: Props ) {
 	const translate = useTranslate();
 
-	const isMultipleEmailEnabled: boolean = isEnabled(
-		'jetpack/pro-dashboard-monitor-multiple-email-recipients'
+	const isDowntimeMonitoringPaidTierEnabled = isEnabled(
```

Comment:
Since we are in the context of downtime monitoring, we could 

```suggestion
	const isPaidTierEnabled = isEnabled(
```